spark/v3.4/spark/src/test/java/org/apache/iceberg/spark/TestSparkDistributionAndOrderingUtil.java

```diff
@@ -1805,23 +1805,32 @@ public void testRangeCopyOnWriteMergePartitionedSortedTable() {
   // -------------------------------------------------------------------------
   // delete mode is NOT SET -> CLUSTER BY _spec_id, _partition, _file +
   //                           LOCALLY ORDERED BY _spec_id, _partition, _file, _pos
+  // delete mode is NOT SET (fanout) -> CLUSTER BY _spec_id, _partition, _file + empty ordering
```

Comment:
Probably annoying to add another example with ORDERED BY, but is it still correct with  "ORDERED" above?    Will it not be empty_ordering?

Reply:
If there is a table sort order, we still respect it by default and fanout writer config has no effect.

Reply:
Yea I meant , in this comment as it is ORDERED & UNORDERED, fanout should not be 'empty ordering'?

Reply:
Oh I see. ok